pkgs/games/gnujump/default.nix

```diff
@@ -11,6 +11,88 @@ stdenv.mkDerivation rec {
 
   NIX_LDFLAGS = "-lm";
 
+  postInstall = ''
+    mkdir -p $out/share/applications $out/share/pixmaps
+  
+    cat > $out/share/applications/gnujump.desktop <<EOF
+    [Desktop Entry]
```

Comment:
There's an undocumented Nix function made for this: `makeDesktopItem`.

Reply:
Thank you for the pointer, back then I didn't know about makeDesktopItem. Done!